src/components/UserInputComponent/UserInputForm.js

```diff
@@ -29,7 +29,7 @@ function UserInputForm(props) {
       setOpen(false);
     };
     const handleTooltipOpen = () => {
-      setOpen(true);
+      setOpen((r) => !r);
```

Comment:
```suggestion
      setOpen((prevValue) => !prevValue);
```
I know I told you to just write `setOpen((r) => !r)`, but it could be changed to be clearer :D